packages/inventory/src/types.ts

```diff
@@ -1,9 +1,8 @@
+import { ItemRowFragment } from '@openmsupply-client/system';
 import { StocktakeLineFragment } from './Stocktake/api/operations.generated';
 
 export interface StocktakeSummaryItem {
   id: string;
-  itemId: string;
-  itemName: string;
-  itemCode: string;
+  item?: ItemRowFragment | null;
```

Comment:
For some reason a stocktake line has an `Item` as optional, which is a little odd!